src/custom/float.rs

```diff
@@ -46,18 +50,18 @@ impl<'c, H: HugrView> CodegenExtension<'c, H> for FloatTypesCodegenExtension {
         }
     }
 
-    fn emitter<'a>(
+    fn emit_extension_op<'c>(
         &self,
-        _context: &'a mut crate::emit::func::EmitFuncContext<'c, H>,
-    ) -> Box<dyn crate::emit::EmitOp<'c, hugr::ops::ExtensionOp, H> + 'a> {
-        Box::new(NullEmitLlvm)
+        _: &mut EmitFuncContext<'c, H>,
+        args: EmitOpArgs<'c, '_, ExtensionOp, H>,
+    ) -> Result<()> {
+        bail!("Unsupported op: {}", args.node())
```

Comment:
It might be nice to say something bespoke like "FloatTypes doesn't implement any custom ops, try FloatOps"